docker-compose.travis.yml

```diff
@@ -1 +1,27 @@
 version: '3.7'
+
+services:
+  web:
+    environment:
+      CELERY_TASK_ALWAYS_EAGER: 'True'
+      SECRET_KEY: 'not_very_secret_key'
+      OPEN_DISCUSSIONS_BASE_URL: 'http://localhost:8063/'
+      OPEN_DISCUSSIONS_COOKIE_NAME: 'cookie_monster'
+      OPEN_DISCUSSIONS_COOKIE_DOMAIN: 'localhost'
+      OPEN_DISCUSSIONS_DEFAULT_SITE_KEY: 'mm_test'
+      MAILGUN_SENDER_DOMAIN: 'other.fake.site'
+      MAILGUN_KEY: 'fake_mailgun_key'
+      ELASTICSEARCH_INDEX: 'testindex'
+      INDEXING_API_USERNAME: 'mitodl'
+  celery:
+    environment:
+      CELERY_TASK_ALWAYS_EAGER: 'True'
+      SECRET_KEY: 'not_very_secret_key'
+      OPEN_DISCUSSIONS_BASE_URL: 'http://localhost:8063/'
+      OPEN_DISCUSSIONS_COOKIE_NAME: 'cookie_monster'
+      OPEN_DISCUSSIONS_COOKIE_DOMAIN: 'localhost'
+      OPEN_DISCUSSIONS_DEFAULT_SITE_KEY: 'mm_test'
+      MAILGUN_SENDER_DOMAIN: 'other.fake.site'
+      MAILGUN_KEY: 'fake_mailgun_key'
+      ELASTICSEARCH_INDEX: 'testindex'
+      INDEXING_API_USERNAME: 'mitodl'
```

Comment:
Can we not use `x-environment` to specify all of these and reuse them like `docker-compose.yml` does?